.github/workflows/rust.yml

```diff
@@ -37,6 +37,7 @@ jobs:
             ~/.cargo/git/db/
             target/
       - run: cargo test --workspace --all-targets
+      - run: cargo run --example get_timezone
```

Comment:
I don't know how to change the timezone of the runner, so don't know if there is a better option than to at the result whether it looks sane.